src/posts/posts-api.ts

```diff
@@ -12,6 +12,13 @@
     }
   };
 
+  export const getPublishedPosts = async () => {
+    await sleep(500);
+    const storedPosts = localStorage.getItem(LOCAL_STORAGE_KEY);
+    const posts: Post[] = storedPosts ? JSON.parse(storedPosts) : [];
+    return posts.filter((post) => post.status === "published");
```

Comment:
lets use string enum to replace status strings, `PostStatusesEnum.PUBLISHED` , this approach allow us to use centralized values for statuses, so if the status value changed, we can change it only in one place  (which you declare your enum)